manifests/s/SpecialK/SpecialK/25.2.6/SpecialK.SpecialK.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: SpecialK.SpecialK
+PackageVersion: 25.2.6
+InstallerType: inno
+UpgradeBehavior: install
+ProductCode: '{F4A43527-9457-424A-90A6-17CF02ACF677}_is1'
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://sk-data.special-k.info/repository/SpecialK_25.2.6.exe
+  InstallerSha256: EBFCFD5E78662DC4D8C5B29EB0BF874388129E9D01ABC2B62F494D3428F7162F

```

```suggestion
 InstallerSha256: 409B54CD0919773155E6CDB4B1FB305E1727744EB89BF4388EF2B2FA7CD5297C
```

(Automated response - build 977.)